### ISSUE TITLE
Quotes when 'backgroundColor' style

### ISSUE THREAD

**Issue by aloisdeniel** [dart-browser/universal_html#9]
When I define a color value to the property `backgroundColor`, additionnal quotes are printed in outputed html.

```
final div = DivElement();
div.style.backgroundColor = "#FF8800";
print(div.outerHtml); // <div style="background-color: &quot;#FF8800&quot;"></div>
```

**Comment by terrier989** [comment]
Hi, thanks for the report. I fixed it and published a new version (1.1.12).

**Comment by aloisdeniel** [comment]
Thank you for the quick update!

Oh and I didn't mentioned it, but it happened also with `backgroundCompound` property.
